RPIPs/RPIP-44.md

```diff
@@ -1,56 +0,0 @@
-
-## Specification
-
-This specification extends the specification of megapools in
-[RPIP-43](RPIP-43.md) with a facility for removal of validators from the
-megapool to be done by parties other than the Node Operator.
-
-Funds (ETH) associated with a megapool SHALL be accounted in (at least) the following categories:
-- Received Funds (`received`): Funds provided that can be used towards validator deposits; sources include deposits from the Node Operator and ETH staked on behalf of the node ([RPIP-32](RPIP-32.md))
-- Credits (`credit`): Funds already in the protocol that can be used towards validator deposits; sources include ETH already staked on the beacon chain for a migrating validator
-- Deposits (`deposited`): Funds currently staked on the beacon chain for validators associated with this megapool
-- Withdrawals (`withdrawn`): Funds received into the megapool via withdrawals from the beacon chain, including both principal and the node's share of rewards, NOT including pool stakers' share of the rewards
-- Debt (`debt`): Penalties or shortfall from exited validators (see [RPIP-43](RPIP-43.md))
-  
-The following quantity, `deficit`, is derived from the categorised funds above:
-```math
-$$ \mathtt{deficit} = \mathtt{debt} - \mathtt{received} - \mathtt{credit} -  \mathtt{withdrawn} $$
-```
-
-We also introduce a protocol-wide parameter `exit_deficit`, with initial setting
-```math
-$$ \mathtt{exit\_deficit} = 0.2\;\mathrm{ETH} $$
-```
-
-The exit functionality for a megapool is specified as follows:
-- A megapool contract SHALL include an `exit` function
-  - This function SHALL accept one or more public keys for validators associated with this megapool
-  - This function SHALL either remove the specified validators from the megapool or revert
-      - If any of the validators are still active on the beacon chain a successful call of this
-        function MUST also exit those validators from the beacon chain
-  - This function MAY be freely called by the Node Operator (as implied by [RPIP-43](RPIP-43.md))
-  - This function MAY be called by any account under the condition: `deficit >= exit_deficit`
-      - It MUST NOT be possible for accounts other than the Node Operator to exit more validators than needed to reduce `deficit` below `exit_deficit`
-- The protocol SHALL use the `withdrawn`, `credit`, and `received` balances to pay off `debt` prior to taking action on exits
-- The protocol SHOULD use staked RPL to decrease `debt` by the corresponding amount prior to taking action on exits
-
```

Comment:
Needs rationale and security considerations.

Reply:
Security considerations exist now; still needs rationale